raiden/tests/scenarios/mfee1_flat_fee.yaml

```diff
@@ -0,0 +1,73 @@
+version: 2
+
+settings:
+  gas_price: "fast"
+  chain: any
+  services:
+    pfs:
+      url: https://pfs-goerli-with-fee.services-dev.raiden.network
+    udc:
+      enable: true
+      token:
+        deposit: true
+
+token:
+  address: "0x62083c80353Df771426D209eF578619EE68D5C7A"
+
```

Comment:
Could you please add `balance_min: 1_000_000` here then I think we can merge it.

Reply:
Why do I need to do that? Shouldn't the SP mint enough tokens?

Reply:
No, right now it's hard coded to only mint 5000 if nothing else is specified.

Reply:
Ok, did the changes!